src/test/java/racingcar/dao/JdbcPlayerDaoTestEntity.java

```diff
@@ -0,0 +1,94 @@
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.transaction.annotation.Transactional;
+import racingcar.controller.ApplicationType;
+import racingcar.entity.GameEntity;
+import racingcar.entity.PlayerEntity;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+@JdbcTest
+@Transactional
+class JdbcPlayerDaoTestEntity {
+
+    private PlayerDao playerDao;
+    private GameDao gameDao;
+
+    @Autowired
+    private NamedParameterJdbcTemplate jdbcTemplate;
+
+    @BeforeEach
+    void setUp() {
+        playerDao = new JdbcPlayerDao(jdbcTemplate);
+        gameDao = new JdbcGameDao(jdbcTemplate);
+    }
+
+    @Test
+    @DisplayName("Player 테이블에 값을 저장할 수 있다.")
+    void savePlayer_whenCall_thenSuccess() {
+        // given
+        Long gameId = gameDao.save(new GameEntity(10, ApplicationType.CONSOLE));
+        final PlayerEntity kongHana = new PlayerEntity("콩하나", 10, true);
+        final PlayerEntity ethan = new PlayerEntity("에단", 5, false);
+
+        // when, then
+        assertThatCode(() -> playerDao.save(gameId, List.of(kongHana, ethan)))
+                .doesNotThrowAnyException();
+    }
+
+    @Test
+    @DisplayName("Game 테이블에 저장되지 않은 gameId를 사용하면 Player 테이블에 값을 저장할 수 없다.")
+    void savePlayer_whenInvalidGameId_thenFail() {
+        // given
+        final PlayerEntity kongHana = new PlayerEntity("콩하나", 10, true);
+        final PlayerEntity ethan = new PlayerEntity("에단", 5, false);
+
+        // when, then
+        assertThatThrownBy(() -> playerDao.save(2, List.of(kongHana, ethan)))
+                .isInstanceOf(DataAccessException.class);
+    }
+
+    @Test
+    @DisplayName("player 테이블에 저장된 정보를 불러올 수 있다.")
+    void findPlayer_whenCall_thenSuccess() {
+        // given
+        String konghanaName = "콩하나";
+        String ethanName = "에단";
+
+        // when
+        Long gameId = gameDao.save(new GameEntity(10, ApplicationType.CONSOLE));
+        final PlayerEntity kongHana = new PlayerEntity(konghanaName, 10, true);
+        final PlayerEntity ethan = new PlayerEntity(ethanName, 5, false);
+        playerDao.save(gameId, List.of(kongHana, ethan));
+
+        List<PlayerEntity> playerEntity = playerDao.findById(gameId);
+
+        // then
+        Assertions.assertAll(
+                () -> assertThat(playerEntity.size()).isEqualTo(2),
+                () -> assertThat(playerEntity.get(0).getName()).isEqualTo(konghanaName),
+                () -> assertThat(playerEntity.get(0).getPosition()).isEqualTo(10),
+                () -> assertThat(playerEntity.get(0).getIsWinner()).isEqualTo(true),
+                () -> assertThat(playerEntity.get(1).getName()).isEqualTo(ethanName),
+                () -> assertThat(playerEntity.get(1).getPosition()).isEqualTo(5),
+                () -> assertThat(playerEntity.get(1).getIsWinner()).isEqualTo(false)
```

Comment:
이 부분은 given을 아래처럼 주었다면 [recursiveComparison](https://stackoverflow.com/questions/43316133/how-to-compare-recursively-ignoring-given-fields-using-assertj) 이란 문법도 활용할 수 있습니다.
```java
List<PlayerEntity> givenEntities = Arrays.asList(new PlayerEntity(konghanaName, 10, true), new PlayerEntity(ethanName, 5, false));
```

```suggestion
             () -> assertThat(given).usingRecursiveComparison()
            .ignoringFields("id", "gameId")
            .isEqualTo(playerEntity);
```